src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -299,6 +308,15 @@ const WorkspaceListTable = () => {
               },
               onCancel: () => navigate({ pathname: pathnames.workspaces.link }),
             },
+            [pathnames['edit-workspaces-list'].path]: {
+              afterSubmit: () => {
+                dispatch(fetchWorkspaces());
+                navigate(mergeToBasename(paths.workspaces.link.replace(':workspaceId', editWorkspace?.id ?? '')));
```

Comment:
Do not use `mergeTobasename` with link, it will break the flow. If user submits or cancels the edit they will be redirected to `iam/user-access/iam/user-access/workspaces/:ID`.